mlflow/gateway/config.py

```diff
@@ -34,7 +34,7 @@ class RouteType(str, Enum):
     LLM_V1_EMBEDDINGS = "llm/v1/embeddings"
 
 
-class CohereConfig(BaseModel, extra=Extra.allow):
+class CohereConfig(BaseModel):
```

Comment:
`extra=Extra.ignore` is the default behavior, but explicitly defining `extra=Extra.ignore` appears to be incompatible with a `Config` nested class that we use to define docs schema examples with `schema_extra`. Accordingly, I'm omitting `extra=Extra.ignore` in favor of the implicit behavior